NestJS/src/Inbox/inbox.service.ts

```diff
@@ -20,14 +20,15 @@ export class InboxService {
       await this.inboxRepository.update(toUpdate.id,toDate);
       return toUpdate;
     }catch (e){
-      console.log(e)
+      return e;
```

Comment:
this is persistent operation, so in case of failed actions it should not just return error, raising will be more suitable